tests/cupyx_tests/scipy_tests/signal_tests/test_signaltools.py

```diff
@@ -18,23 +18,17 @@ class TestChooseConvMethod(unittest.TestCase):
     @testing.for_dtypes('efdFD')
     @testing.numpy_cupy_equal(scipy_name='scp')
     def test_choose_conv_method1(self, xp, scp, dtype):
-        a = testing.shaped_arange((10,), xp, dtype)
-        b = testing.shaped_arange((5,), xp, dtype)
+        a = testing.shaped_arange((10000,), xp, dtype)
+        b = testing.shaped_arange((5000,), xp, dtype)
         return scp.signal.choose_conv_method(a, b, mode=self.mode)
```

Comment:
`cupy.choose_conv_method` supported in this PR does not guarantee the compatibility to SciPy's one. So please test it without `numpy_cupy_equal`.